Front-End/challenge1/script.js

```diff
@@ -1 +1,16 @@
 // Modify this file only
+const decrease = document.getElementById('decrease');
+const increase = document.getElementById('increase');
+const counter = document.getElementById('counter');
+
+function counting(nro) {
+  let current = +(counter.innerText);
```

Comment:
why did you wrap in parenthesis this `counter.innerText)`?

Reply:
Well I thought I needed to isolate the value, but I realize that it is unnecessary. Thanks